src/focus_manager/txm_platform.js

```diff
@@ -370,6 +379,8 @@ export class TXMPlatform {
             const versionMatch = userAgent.match(/FW\/([^\s)]+)/);
             self.version = versionMatch && versionMatch[1] || "?.?.?";
 
+            self.isSlowDevice = !!self.model.match(/^D24/); // D-series has bad performance
```

Comment:
I think 24 means it is 24 inch. May be just `^D[0-9]`?

Reply:
To be fair, I only know that the D24 is slow, nothing else.

OTOH, it turns out that I never needed to make use of this flag, since I never had to actually distinguish between the fast vs slow vizio models.